src/main/java/io/codearte/jfairy/producer/person/locale/ge/GePassportNumberProvider.java

```diff
@@ -0,0 +1,36 @@
+package io.codearte.jfairy.producer.person.locale.ge;
+
+import io.codearte.jfairy.producer.BaseProducer;
+import io.codearte.jfairy.producer.person.PassportNumberProvider;
+
+import javax.inject.Inject;
+
+public class GePassportNumberProvider implements PassportNumberProvider {
+
+	private final BaseProducer baseProducer;
+
+	@Inject
+	public GePassportNumberProvider(BaseProducer baseProducer) {
+		this.baseProducer = baseProducer;
+	}
+
+	// District codes are not reachable at the moment.
```

Comment:
What does this mean that they are not reachable? Is there any list of codes or random numbers are being used?

Reply:
I could not find the codes anywhere, GE Ministry of Interior has not published them.